tools/backend-webpack/config/getWebpackConfig.js

```diff
@@ -35,6 +35,9 @@ function getWebpackConfig(srcFile, outDir, profile) {
       // See https://github.com/facebook/create-react-app/issues/240
       new CaseSensitivePathsPlugin(),
     ],
+    externals: {
+      "encoding": "require('encoding')",
```

Comment:
This should probably go here instead: https://github.com/iTwin/itwinjs-core/blob/release/2.19.x/tools/webpack-core/src/plugins/BackendDefaultsPlugin.ts#L68

Reply:
Actually, try adding `"node-fetch"` to here instead: https://github.com/iTwin/itwinjs-core/blob/release/2.19.x/tools/webpack-core/src/plugins/BackendDefaultsPlugin.ts#L49

Reply:
ooh, so I was adding the wrong package to the optional dependencies plugin... by the way, I find it weird that isn't the default for webpack. If someone wraps a require in a try/catch, it's obvious they _expect_ it to not always be available.